Client/Frontend/Home/LogoHeader/HomeLogoHeaderViewModel.swift

```diff
@@ -58,7 +58,7 @@ extension HomeLogoHeaderViewModel: HomepageViewModelProtocol, FeatureFlaggable {
 
         let leadingInset = HomepageViewModel.UX.leadingInset(traitCollection: traitCollection)
         section.contentInsets = NSDirectionalEdgeInsets(
-            top: height * factor,
+            top: 24,
```

Comment:
Can this be move to the `UX` struct so it's more visible? And shouldn't this only be applied for when the experiment is turned on?

Reply:
The resulting value is basically that one but will definitely make it "experiment dependant", I agree